cmd/minikube/cmd/logs.go

```diff
@@ -30,14 +30,16 @@ import (
 // logsCmd represents the logs command
 var logsCmd = &cobra.Command{
 	Use:   "logs",
-	Short: "Gets the logs of the running OpenShift instance, used for debugging minishift, not user code.",
-	Long:  `Gets the logs of the running OpenShift instance, used for debugging minishift, not user code.`,
+	Short: "Gets the logs of the running Minishift instance.",
+	Long:  `Gets the logs of the running Minishift instance and prints them to the standard output. You use these log messages to debug Minishift.
```

Comment:
This text was already there, and I didn't remove it. My guess would be that we should differentiate what these logs are for, since we're dealing with Minishift>VM>OpenShift>app.